depscan/cli.py

```diff
@@ -593,10 +594,18 @@ async def run_scan():
     bom_file_path = None
 
     if uploaded_bom_file.get('file', None) is not None:
+        bom_file_content = uploaded_bom_file['file'].read().decode('utf-8')
+        try:
+            _ = json.loads(bom_file_content)
+            _ = parse(bom_file_content)
```

Comment:
I'm not sure I understand this. Are we parsing the same content as json and then XML?

Reply:
Fixed this.